events/janus_rabbitmqevh.c

```diff
@@ -99,6 +102,17 @@ static amqp_channel_t rmq_channel = 0;
 static amqp_bytes_t rmq_exchange;
 static amqp_bytes_t rmq_route_key;
 
+static char *rmqhost = NULL;
+static const char *vhost = NULL, *username = NULL, *password = NULL;
+static const char *ssl_cacert_file = NULL;
+static const char *ssl_cert_file = NULL;
+static const char *ssl_key_file = NULL;
+static gboolean ssl_enable = FALSE;
+static gboolean ssl_verify_peer = FALSE;
+static gboolean ssl_verify_hostname = FALSE;
+static const char *route_key = NULL, *exchange = NULL, *exchange_type = NULL ;
+static uint16_t heartbeat = 0;
```

Comment:
Shouldn't there be a reasonable default? I assume `0` disables it, meaning by default it's not used?

Reply:
With the heartbeat equals to 0 the heartbeat monitoring process is disabled. Was decided to default to 0 to maintain the behavior previously existent on the event handler.